.github/test.yml

```diff
@@ -0,0 +1,27 @@
+name: Lint & Test Workflow
+
+on:
+  pull_request:
+    branches: [main, dev]
+
+jobs:
+  build:
```

Comment:
between lines 7 and 8 there should be a job name that will group all of the steps under eg:
jobs:
    

```yaml
jobs:
    run_tests:
        build:
           <the rest>
```

Reply:
I see what you mean, for this workflow though we have the name of the overall workflow as a title (at the top of the file) and then each step has a name. I think as it is, that works ok, as they're all tied together by the title name. We could always add names for jobs too, if we create more jobs at a later date, but for now I think it's ok without. 

What do you think?

Reply:
I like the idea of having one test per file, however if we have many workflows it will spin up a new server for each file, and that will take some time, for now I believe that is fine but we'll see at a later date if we require to change the yaml structure to have the name of the test inside the jobs property

Reply:
@marktlinn I will mark this as resolved however, I'd rather have the person who creates the PR to be the one who merges it, so I will leave that to you